### ISSUE TITLE
1.0.0 release

### ISSUE THREAD

**Issue by adamralph** [liteguard/liteguard#113]
**Ready** when all other issues forming part of the release are **Done**.
- [x] run code analysis in VS in _Release_ mode and address violations (send a regular PR which must be merged before continuing)
- [x] check build, create release in GitHub UI including releaseNotes, mentioning non-owner contributors, if any
- [x] push nuget package
- [x] tweet, mentioning contributors and post link as comment here for easy retweeting ;-)
- [x] post tweet in https://gitter.im/liteguard/liteguard
- [x] post links to the Tweet, NuGet and GitHub release in each issue in this milestone, with thanks to contributors
- [x] increment minor version
- [x] push to origin branch, create PR to upstream master
- [x] check build, merge PR
- [x] create a new milestone for the next release
- [x] create new issue based on this one for next release, adding it to the new milestone
- [x] close all issues on this milestone
- [x] close this milestone

**Comment by adamralph** [comment]
https://twitter.com/adamralph/status/795979884094955521
